### PR TITLE
Cleanup unused emitter arguments

### PR DESCRIPTION
**Pull request by briansull**
Also removed unused idClsCookie from struct instrDescDebugInfo

### CODE REVIEW

**Comment by briansull** [comment]
@dotnet-bot Test Windows_NT arm Cross Checked Build and Test
@dotnet-bot Test Tizen armel Cross Debug Build

**Comment by briansull** [comment]
@dotnet-bot Test Windows_NT arm Cross Checked Build and Test

**Comment by briansull** [comment]
@dotnet/jit-contrib PTAL

File: src/jit/emit.cpp [hunk 1]
```diff
@@ -1337,7 +1337,7 @@ void* emitter::emitAllocInstr(size_t sz, emitAttr opsz)
 
     emitInsCount++;
 
-#if defined(DEBUG) || defined(LATE_DISASM)
```
Comment (BruceForstall) [inline src/jit/emit.cpp]: Can you make sure a RELEASE build with LATE_DISASM defined still builds and works?

Comment (briansull) [reply src/jit/emit.cpp]: OK, I tried defining LATE_DISASM and it gets 11 unresolved symbols all related to DIS.
I suppose it I actually tried linking the disassembler library it work work.

Comment (briansull) [reply src/jit/emit.cpp]: Ok linking also works if I change jit.nativeproj:
    <EnableLateDisasm Condition="'$(BuildArchitecture)' != 'arm' and '$(BuildProjectName)' != 'CoreSys'">true</EnableLateDisasm>

**Pull request merged**